### ISSUE TITLE
ShaderGlass in full-screen not always above other windows

### ISSUE THREAD

**Issue by somatronicon** [rohatsu/ShaderGlass#12]
In some cases, I've noticed the full-screen window doesn't stay on top of other windows. For example, if I click on the date/time in the system tray, it pops up in ShaderGlass with the shader applied, and also above the full-screen ShaderGlass window without the shader applied. This also happens with the start menu, tooltips, context menus, etc.

I don't know if there is a way in the Win32 API to make the ShaderGlass FS window actually completely cover all other Windows; that'd be mint though if it could.

Thanks again for the great project!

**Comment by rohatsu** [comment]
Yes I think staying on top of Windows and all of its popups (Action Center etc.) is something even full-screen games struggle with, it would require constant monitoring/resetting of window order. I haven't really envisioned ShaderGlass covering the whole desktop, more as an overlay for emulators and old games, but I'll keep this on the list next time I sit down.

**Comment by somatronicon** [comment]
No problem, and I can definitely see how this would be a bit more difficult. I did a bit of digging and stumbled onto the following information:

https://stackoverflow.com/a/13497452

![image](https://user-images.githubusercontent.com/86700194/124007475-03b3a900-d9a1-11eb-9a9a-60330e73a93b.png)

I also found a couple other sites related to Windows 8 that might help with something like this. Basically the same thing as the screenshot above, but it gives more information about signing, etc.

https://pixytech.com/2013/05/01/windows-8-topmost-window/

The archived version has images:

https://web.archive.org/web/20200929140314/https://pixytech.com/2013/05/01/windows-8-topmost-window/

Thanks for considering it! I totally understand that this wasn't part of the original design, so no problem if it doesn't get picked up.

Thanks @rohatsu 👍
